main.py

```diff
@@ -192,10 +208,11 @@ def escape_path(path):
 			return path.replace(',', ',,')
 
 		def genArg(settings, name, port):
-			if settings.settings[name] is not '':
+			port_arr = ['controller_three', 'controller_four', 'controller_one', 'controller_two']
+			if settings.settings[name] is not '' and settings.settings[port_arr[int(port[:1]) - 1]] != 'Not connected':
 				check_path(settings.settings[name])
 				return ['-drive', 'if=none,id=' + name + ',file=' + escape_path(settings.settings[name]),
-				        '-device', 'usb-storage,drive=' + name + ',port=' + port]
+						'-device', 'usb-storage,drive=' + name + ',port=' + port]
```

Comment:
Accidental change.

This is meant to align with line above, so it needs tabs until `return`, then spaces to align the rest.